packages/SwingSet/src/kernel/slogger.js

```diff
@@ -248,6 +256,7 @@ export function makeSlogger(slogCallbacks, writeSlogObject, slogDuration) {
         vatID,
         deliveryNum,
         syscallNum,
+        syscall: ksc?.[0],
```

Comment:
Why not use `vsc` here?

```suggestion
        syscall: vsc?.[0],
```